parser_library/src/workspaces/wildcard.cpp

```diff
@@ -38,5 +42,44 @@ std::regex wildcard2regex(const std::string& wildcard)
     regex_str = std::regex_replace(regex_str, nongreedy, ".$1?");
     return std::regex(regex_str);
 }
+std::set<std::filesystem::path> wildcard_recursive_search(std::string lib_path, int pos)
+{
+    std::set<std::filesystem::path> list_of_libs;
+
+    auto prefix_path = lib_path.substr(0, pos);
+    pos = pos + 1;
+    for (const auto& p : std::filesystem::recursive_directory_iterator(prefix_path))
+    {
+        std::string suffix_path = lib_path.substr(pos);
+        suffix_path.pop_back();
+        suffix_path = std::regex_replace(suffix_path, std::regex("\\*"), ".*");
+
+
+        std::string directory_path = p.path().lexically_normal().generic_string();
```

Comment:
I am not sure this will work correctly in the WASM variant. See utils/include/utils/* and utils/src/* for more details...